Extension/src/LanguageServer/client.ts

```diff
@@ -967,6 +968,8 @@ export class DefaultClient implements Client {
                 }
             }
         }
+        compilerCount = paths.length;
+        telemetry.logLanguageServerEvent('compilerSelection', { action :"compilerCount"}, { compilerCount })
```

Comment:
Can you use `{ compilerCount: paths.length }` without the compilerCount local variable?

Reply:
Also, can you format this? I think `action :` should be "`action: `.

Reply:
These lines are being deleted. It will merge with the other call to log this event

Reply:
But `{ compilerCount: paths.length }` will work without needing a `compilerCount` variable.